README.md

```diff
@@ -149,23 +150,59 @@ the creation of a new management pack project.
 7. `Enter a path to the Management Pack icon file, or leave blank for no icon`
 
    The icon is used in the VMware Aria Operations UI if present. If it is not present, a default icon will be used. The icon file must be
-   png format and 256x256 px. An icon file can be added later by copying the icon to the root project directory and
+   PNG format and 256x256 pixels. An icon file can be added later by copying the icon to the root project directory and
    setting the value of the `"pak_icon"` key to the icon's file name in the `manifest.txt` file.
 
-8. `Select a language for the adapter. Supported languages are [...]`
-
-   Supported languages are listed. Once selected, the project will be generated, including a template adapter in the
-   selected language. The template adapter collects several objects and metrics from the container that the adapter 
-   is running in, and can be used as a starting point for creating a new adapter.
-
-For complete documentation of the `mp-init` tool see the [MP Initialization Tool Documentation](doc/mp-init.md).
+[//]: # (TODO: Add this section back when we support it)
+[//]: # (8. `Select a language for the adapter`)
+[//]: # (  selected language. The template adapter collects several objects and metrics from the container that the adapter)
+[//]: # (  Once selected, the project will be generated, including a template adapter in the)
+[//]: # (  is running in, and can be used as a starting point for creating a new adapter.)
+
+For complete documentation of the `mp-init` tool including an overview of its output, see the [MP Initialization Tool Documentation](doc/mp-init.md).
+
+### Template Adapter
+The template adapter has comments throughout the code to help new users understand the process of creating their own adapter using the existing code. For additional guidance creating adapters see the [Walkthroughs](../README.md#walkthroughs) section.
+The code inside `adapter.py`  has four methods:
```

Comment:
This section starts talking about the template adapter right away without any discussion about that the template adapter is or where it's located or why we should care. Also, we should be careful about talking about things that are specific to the template adapter vs things that are generally true of any adapter. The methods below are in the latter category, so talking about them only in terms of the sample adapter is potentially misleading.